README.md

```diff
@@ -169,17 +175,60 @@ Pretty will output a yaml like content.
 ``` js
     const logger = require('@ekino/logger')
       
+    logger.setNamespaces('namespace:*');
+    logger.setLevel('debug');
+    logger.setOutput(logger.outputs.pretty);
+   
+    const log = logger('namespace:subNamespace');
+    log.debug('ctxId', 'Will be logged',{someData: 'someValue', someData2: 'someValue'});
+```
+output : 
+
+![Example](docs/images/example_pretty.gif)
+
+#### Output function
+An output, is a function that will receive log data and should transform and store it
+
+Log data follow the format : 
+```
+{
+    time: Date,
+    level: string,
+    namespace: string,
+    contextId: string,
+    meta: { any data defined in global context },
+    message: string,
+    data: object
+}
+```
+
+``` js
+   const logger = require('@ekino/logger')
+      
    logger.setNamespaces('namespace:*');
    logger.setLevel('debug');
-   logger.setOutput('pretty');
+   
+   function consoleAdapter(log) {
+      console.log(logger.outputUtils.stringify(log))
+   }
+   
+   logger.setOutput([logger.output.pretty, consoleAdapter]);
```

Comment:
Maybe we should add a comment about what the output will look like.

Reply:
I added a comment